.github/workflows/conformance-aks.yaml

```diff
@@ -20,7 +20,7 @@ on:
   #   will disappear from the PR checks: please provide a direct link to the
   #   successful workflow run (can be found from Actions tab) in a comment.
   # 
-  # pull_request: {}
+  pull_request: {}
```

Comment:
Ah right I guess we should wait for the results of this and revert this change before merging but otherwise LGTM :+1:

Reply:
Yep. I actually intended to run the workflow a few times to see if I can reproduce the install issue over at #18107 and get logs.

Reply:
IMO, we should put those workflow PR in draft until we're finished with testing.